sandbox/src/mocks/getReferralRequest/responses/Unbooked.json

```diff
@@ -0,0 +1,80 @@
+{
+  "id": "000000070000",
+  "meta": {
+    "versionId": "3",
+    "profile": [
+      "http://fhir.nhs.uk/StructureDefinition/ers-referralrequest-1"
+    ]
+  },
+  "resourceType": "ReferralRequest",
+  "contained": [
+    {
+      "id": "Patient-9912003888",
+      "meta": {
+        "profile": [
+          "http://fhir.nhs.uk/StructureDefinition/ers-patient-1"
+        ]
+      },
+      "resourceType": "Patient",
+      "identifier": [
+        {
+          "system": "http://fhir.nhs.uk/Id/nhs-number",
+          "value": "9912003888"
+        }
+      ]
+    },
+    {
+      "id": "DocumentReference-70002",
```

Comment:
Is there a conventional Document Reference number to use, just like NHS number? (I inherited this value from the "unbooked.json" XAPI test I based this example on)

Reply:
Discussed with Hubert. There isn't.